Installation improvements

Luanf opened the pull request:
See my comment [here](https://github.com/Luanf/cligpt/pull/2#issuecomment-1492673131)

If we merge this, I'll merge yours to the head repo.

mehmetalici commented:
Another installation method came to my mind today might be using symbolic links (inspired by installation of https://github.com/Canop/broot today). 

The steps are roughly as follows:
1. Make an executable from cligpt function by removing its function paranthesis and adding a shebang line /bin/bash.
2. On install.sh, we can just say `ln -s ./cligpt $HOME/.local/bin`

Usage would be the same `cligpt bla bla`. 

One downside of this approach is that the user has to add `$HOME/.local/bin` to `$PATH` (if not done yet) and it has to be noted on README.

You can view the approach in https://github.com/mehmetalici/cligpt/tree/install-symbolic-link.

This has also the advantage of removing the `zsh` and `bash` conditional branches in the code as all code will be interpreted by bash regardless of the current shell session thanks to the shebang lines. So, the scripts will be written only for bash.

Luanf commented:
Thank you again for the suggestion! 
I've updated the installation method to use symbolic links as you proposed.
The new approach simplifies the code and is more robust in a compatibility sense.
I've also updated the README to reflect these changes.
Please review the updated code and let me know if you have any further suggestions or concerns.
